invenio_records_lom/cli.py

```diff
@@ -33,6 +34,32 @@ def rebuild_index():
     click.secho("Reindexed records!", fg="green")
 
 
+@lom.command()
+@with_appcontext
+def check():
```

Comment:
minor: normally i would add an `click.option` to check a specific `pid` too. you can keep the loop, but what you put into the loop depends on if a specific check is requested or not.

Reply:
wanted to eventually add one too
no time like today, I guess
added